roles/confluent.variables/filter_plugins/filters.py

```diff
@@ -311,3 +315,48 @@ def c3_ksql_properties(self, ksql_group_list, groups, hostvars, ssl_enabled, htt
                     final_dict['confluent.controlcenter.ksql.' + ansible_group + '.ssl.key.password'] = str(keystore_keypass)
 
         return final_dict
+
+    def resolve_principal(self, common_names:str, rules:str):
```

Comment:
can we get some tests for this?

You could probably just run the filter a few times in a verify.yml

Reply:
Added a few tests. Adding test results here for reference 

```
PLAY [Validate SSL mapping rule] ***********************************************

TASK [Validate mapping rule with Upper case single match] **********************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Validate mapping rule with multiple match] *******************************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

TASK [Validate mapping rule with lowercase single match] ***********************
ok: [localhost] => {
    "changed": false,
    "msg": "All assertions passed"
}

PLAY RECAP *********************************************************************
localhost                  : ok=3    changed=0    unreachable=0    failed=0    skipped=0    rescued=0    ignored=0

```